docs/tutorials/communicator-tutorial.md

```diff
@@ -7,28 +7,41 @@ description: Learn how to communicate with external APIs and services
 In this guide, you'll learn how to communicate with external APIs and services from a DBOS application.
 
 We recommend that all communication with external services be done in _communicator_ functions.
-For example, you can use communicators to serve a file from [AWS S3](https://aws.amazon.com/s3/), call an external API like [Stripe](https://stripe.com/) or access a non-Postgres data store like [Elasticsearch](https://www.elastic.co/elasticsearch/).
+For example, you can use communicators to serve a file from [AWS S3](https://aws.amazon.com/s3/), call an external API like [Stripe](https://stripe.com/), or access a non-Postgres data store like [Elasticsearch](https://www.elastic.co/elasticsearch/).
 Encapsulating these calls in communicators is especially important if you're using [workflows](./workflow-tutorial) so the workflow knows to execute them only once.
 
 Communicators must be annotated with the [`@Communicator`](../api-reference/decorators#communicator) decorator and must have a [`CommunicatorContext`](../api-reference/contexts#communicatorcontext) as their first argument.
 As with other DBOS functions, communicator inputs and outputs must be serializable to JSON.
-Here's a simple example using [Axios](https://axios-http.com/docs/intro) to call the [Postman Echo API](https://learning.postman.com/docs/developer/echo-api/):
+Here's a simple example using [`fetch()`](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch) to retrieve the contents of `https://example.com`:
 
 
 ```javascript
   @Communicator()
-  static async greetPostman(ctxt: CommunicatorContext, greeting: string) {
-    await axios.get("https://postman-echo.com/get", {
-      params: {
-        greeting: greeting
-      }
-    });
-    ctxt.logger.info(`Greeting sent to postman!`);
+  static async exampleCommunicator(ctxt: CommunicatorContext) {
+    return await fetch("https://example.com").then(r => r.text());
   }
 ```
 
-### Retries
+### Configurable Retries
```

Comment:
I think it might be clearer to add some reliability guarantees of communicators in this tutorial. This is the place we can say, once a communicator function completes, DBOS will never re-execute again. If it throws an exception, DBOS automatically retries based on the configured retry policy.